packages/app-project/src/screens/ClassifyPage/ClassifyPage.js

```diff
@@ -34,7 +34,7 @@ function ClassifyPage (props) {
 
       <Box as='aside' gap='medium'>
         <FinishedForTheDay />
-        {false && <Grid columns={['auto', 'auto', 'auto']} gap='medium'>
+        {<Grid columns={['1fr', '2fr']} gap='medium'>
```

Comment:
Did you mean to remove the `false` check here? If so, the curly braces can be removed too.

Reply:
Good catch. That `false` is a hack to remove the stats box from production.